### PR TITLE
MOTECH-2793: Jasmine tests for i18n

### PR DESCRIPTION
**Pull request by afijal**

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by jredlarski** [comment]
test this please

File: src/i18n/i18n.form.directive.spec.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+describe('i18n form', function() {
+    var element, scope, compile;
+
+  // Store references to $rootScope and $compile
```
Comment (jredlarski) [inline src/i18n/i18n.form.directive.spec.js]: @afijal Remove comments

File: src/i18n/i18n.form.directive.spec.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+describe('i18n form', function() {
```
Comment (jredlarski) [inline src/i18n/i18n.form.directive.spec.js]: @afijal Change to `'i18n form directive'`

**Comment by jredlarski** [comment]
@afijal Additionally, there is a controller for `i18n.form.directive.js` which should have some tests

**Comment by jredlarski** [comment]
test this please

**Pull request merged**